integration-tests/plugins/__tests__/cart/store/index.ts

```diff
@@ -125,14 +125,18 @@ describe("/store/carts", () => {
       await dbConnection.manager.save(priceList1)
 
       const ma_sale_1 = dbConnection.manager.create(MoneyAmount, {
-        variant_id: prodSale.variants[0].id,
         currency_code: "usd",
         amount: 800,
         price_list_id: "pl_current",
       })
 
       await dbConnection.manager.save(ma_sale_1)
 
+      await dbConnection.manager.query(
```

Comment:
I don't put more comments on the test for now until we discuss the result of your tests